### PR TITLE
UG added glossary and updated some examples

### PR DESCRIPTION
**Pull request by ReaganTan00**
Update UG:
* Update images
* Give the example format for certain images
* Link back to TOC at end of a subsection
* Add a glossary

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -50,6 +52,8 @@ This user guide will help you get started and understand how FABook can **seamle
 5. **Contact Information**: The full contact information of a client appears here.
 6. **Data Storage**: This is where the data is stored and loaded from.
 
+[Return to Table of Contents](#table-of-contents)
```
Comment (eugenetayyj) [inline docs/UserGuide.md:55]: why thoo

File: docs/UserGuide.md [hunk 2]
```diff
@@ -91,6 +95,25 @@ This user guide will help you get started and understand how FABook can **seamle
 
  </div>
 
+[Return to Table of Contents](#table-of-contents)
+
+## Glossary
```
Comment (eugenetayyj) [inline docs/UserGuide.md:100]: shld glossary be at the btm?

**Review by eugenetayyj — COMMENTED** [review]
need to settle merge conflicts i think merge conflict for images u shld take my files baa cuz i annotated them alr

**Review by fungusta — CHANGES_REQUESTED** [review]
Maybe for the images you can highlight the important areas like how eugene did

**Pull request closed without merging**